pkg/groups/groups.go

```diff
@@ -27,11 +27,41 @@ func GetGroups(
 	groupCoordinators := []GroupCoordinator{}
 
 	for _, kafkaGroupInfo := range listGroupsResp.Groups {
+
+		topicsList := []string{}
+		topicsMap := map[string]bool{}
+
+		describeGroupsRequest := kafka.DescribeGroupsRequest{
+			GroupIDs: []string{kafkaGroupInfo.GroupID},
+		}
+
+		describeGroupsResponse, err := connector.KafkaClient.DescribeGroups(ctx, &describeGroupsRequest)
+		if err != nil {
+			log.Errorf("Cannot list topics for group :%s \n Error in describing group : %s", kafkaGroupInfo.GroupID, err)
+		} else {
+			if len(describeGroupsResponse.Groups) != 1 {
```

Comment:
I just followed GetGroupDetails implementation: https://github.com/segmentio/topicctl/blob/master/pkg/groups/groups.go#:~:text=if%20len(,%7D
But here in GetGroups, since we are using single groupId at a time and valid groupId (which we got from ListGroups), I think we'll never arrive at a condition with response length other than 1.